custom_components/pfsense/pypfsense/__init__.py

```diff
@@ -594,6 +594,14 @@ def arp_get_mac_by_ip(self, ip, do_ping=True):
             return None
         return response
 
+    def flush_states(self):
```

Comment:
`filter_flush_state_table(self)`

Reply:
per my previous comments, state_table doesn't appear to be a current term in freebsd. should be states to match the underlying pf library

Reply:
how about filter_flush_states? more concise than state table, and still accurate to freebsd

Reply:
Since we last interacted, I discovered the need for a kill states method. It's working great on my end, and with the second service, I think the naming convention makes more sense for `flush_states `and `kill_states`